migrations/versions/08894082221a_replace_collection_name_with_observer_.py

```
@@ -213,31 +220,31 @@ def upgrade() -> None:
                 UPDATE {schema}.documents d
                 SET
                     observer = c.observer,
                     observed = c.observed
                 FROM {schema}.collections c, batch
-                WHERE d.ctid = batch.ctid
+                WHERE d.id = batch.id
                     AND d.collection_name = c.name
                     AND d.peer_name = c.peer_name
                     AND d.workspace_name = c.workspace_name
                     AND (d.observer IS NULL OR d.observed IS NULL)
             """
```

Ruff (0.14.2)

Possible SQL injection vector through string-based query construction

(S608)